### DISCUSSION TITLE
Move xtable and network to Imports?

### DISCUSSION THREAD

**Discussion by TomSnijders** [stocnet/rsiena#22] [General]
Package xtable is used for functions effectsDocumentation and (of course) xtable.sienaFit. 
Package network is used for sienaGOF. Perhaps also for sienaDataCreate; but I'm not sure. Does anybody know? 

But these packages are now both mentioned only as "Suggests". Wouldn't it be better to have them under "Imports"? 
(If network is used in sienaDataCreate, then of course this should be the case.)

**Comment by jhollway** [comment]
On the tables, I wonder whether it might be an idea to offer siena model support to other popular tabulating packages, such as [stargazer](https://cran.r-project.org/web/packages/stargazer/vignettes/stargazer.pdf)?

**Reply by TomSnijders** [reply]
Might be a good idea.

What are the requirements for stargazer to use a results object? The basic stuff now is in the components ans$theta, ans$se, ans$fixed, ans$requestedEffects$effectName. What kind of additional structure would be required for stargazer?
